test_scripts/API/AlertManeuver/015_ATF_Navi_IsReady_no_resp_AlertManeuver_Navi_AlertMan_UNSUP_RES.lua

```diff
@@ -0,0 +1,174 @@
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_InitHMI_OnReady()
+  testCasesForNavi_IsReady.InitHMI_onReady_without_Navi_IsReady(self, 1)
+  EXPECT_HMICALL("Navigation.IsReady")
+  -- Do not send HMI response of Navigation.IsReady
+end
+
+function Test:Precondition_connectMobile()
+  self:connectMobile()
+end
+
+function Test:Precondition_StartSession()
+  self.mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+  self.mobileSession:StartService(7)
+end
+
+commonSteps:RegisterAppInterface("Precondition_RegisterAppInterface")
+
+function Test:Precondition_ActivationApp()
+  local request_id = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.applications[config.application1.registerAppInterfaceParams.appName]})
+  EXPECT_HMIRESPONSE(request_id)
+  :Do(function(_,data)
+    if (data.result.isSDLAllowed ~= true) then
+      local request_id1 = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"DataConsent"}})
+      EXPECT_HMIRESPONSE(request_id1)
+      :Do(function(_,_)   
+        self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality", {allowed = true, source = "GUI", device = {id = config.deviceMAC, name = ServerAddress}})
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,data1)
+          self.hmiConnection:SendResponse(data1.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"}) 
+end
+
+commonSteps:PutFile("Precondition_PutFile", "icon.png")
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_AlerManeuver_Navi_AlertManeuve_UNSUPPORTED_RESOURCE()
+  local cor_id_alertmaneuver = self.mobileSession:SendRPC("AlertManeuver",
+  {ttsChunks =
+      {{
+        text ="FirstAlert",
+        type ="TEXT",
+      },
+      {
+        text ="SecondAlert",
+        type ="TEXT",
+    }},
+    softButtons =
+      {{
+        type = "BOTH",
+        text = "Close",
+        image =
+        {
+          value = "icon.png",
+          imageType = "DYNAMIC",
+        },
+        isHighlighted = true,
+        softButtonID = 821,
+        systemAction = "DEFAULT_ACTION",
+    }}
+  })
+
+  EXPECT_HMICALL("Navigation.AlertManeuver",
+  {
+    softButtons =
+      {{
+        type = "BOTH",
+        text = "Close",
+        image = { imageType = "DYNAMIC"},
+        isHighlighted = true,
+        softButtonID = 821,
+        systemAction = "DEFAULT_ACTION",
+    }},
+    appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+  })
+  :ValidIf(function(_,data)
+    local value_Icon = storagePath .. "icon.png"
+    if (string.match(data.params.softButtons[1].image.value, "%S*" .. "("..string.sub(storagePath, 2).."icon.png)" .. "$") == nil ) then
```

Comment:
@vvvakulenko, based on CI problem with verification of dynamic image, please add:
(data.params.softButtons[1].image.value ~= value_Icon ) in 'if' condition.